PluginBuilder/Controllers/AdminController.cs

```diff
@@ -312,18 +313,21 @@ private async Task SaveEmailSettingsToDatabase(EmailSettingsViewModel model)
         var emailSettingsJson = JsonConvert.SerializeObject(model);
         await conn.SettingsSetAsync("EmailSettings", emailSettingsJson);
     }
-    //
 
     [HttpGet("emailsender")]
     public async Task<IActionResult> EmailSender(string to, string subject, string message)
     {
+        if (!string.IsNullOrEmpty(to) && !IsValidEmailList(to))
+        {
+            TempData[TempDataConstant.WarningMessage] = "Invalid email format in the 'To' field. Please ensure all emails are valid";
+            return RedirectToAction("Plugins");
```

Comment:
This is an invalid redirection... and we also shouldn't redirect to Plugins; rather we should provide in `ModelState.AddModelError` for To field that invalid emails were provided.